be/src/olap/task/engine_cancel_delete_task.cpp

```diff
@@ -53,7 +53,8 @@ OLAPStatus EngineCancelDeleteTask::_cancel_delete() {
     DeleteConditionHandler cond_handler;
     for (TabletSharedPtr temp_tablet : table_list) {
         temp_tablet->obtain_header_wrlock();
-        res = cond_handler.delete_cond(temp_tablet, _request.version, false);
+        del_cond_array* delete_conditions = temp_tablet->mutable_delete_predicate();
```

Comment:
here is not mutablet_delete_predicate()， I think here should call delete_predicates()